docs/sources/get-started/_index.md

```diff
@@ -0,0 +1,78 @@
+---
+description: Get started with the Writer's Toolkit
```

Comment:
```suggestion
description: Get started with Writers’ Toolkit
```
The title is considered a proper noun so the article is extraneous, and was in the plural possessive. I am personally fine with singular possessive.

Reply:
Please change this throughout for consistency.

Reply:
Will do. Thank you for the suggestion.

Reply:
Sorry to disagree that Writer's Toolkit is a proper noun. If I saw a reference to the Writer's Toolkit without the article I would put it back in.  This may be a situation where the rules of usage don't 100% follow the correct rules of grammar, but just a bare "Writer's Toolkit" without an article sounds wrong to my ear.

Reply:
I changed it to Writers' Toolkit throughout the content. I agree that it sounds weird without 'the' article. Also, plural posessive seems weird since I've always written it as singular possessive. At least it's consistent now.

Reply:
> Unfortunately, the original decision wasn't discussed in a public channel but in a multi-person DM that included Brenda, Chris, Eve, and Fiona. I believe it was Ursula who originally proposed it and ensured that the initial publication of Writers' Toolkit had consistent usage.
> 
> In that conversation it was decided that Writers' Toolkit is the name of the project, to quote:
> 
> > Writers' Toolkit - proper noun as it is really very much our thing
> 
> In this sense, it is similar to Grafana Agent or Grafana Mimir.
> 
> I believe it is plural possessive because it is the toolkit for all (many) writers and not just a single writer.
> 
> Writers' Toolkit is definitely our current style but we can open up a roundtable discussion or GitHub issue if we want to discuss that.
> 
> I'm just glad it is no longer the mouthful of "Grafana Technical Writing Contributor's Toolkit" :D

Yes, and my initial input matched @JStickler’s sentiment, where I proposed _Writer’s toolkit_ to match our style and not make it a proper noun. In that case, it would have been "... the _Writer’s toolkit_". But because it was deemed a proper noun, it therefore must not have an article.

Reply:
This is an example of where I agreed to disagree and moved on. Sharing this history here for context about the decision.